src/Compilers/CSharp/Test/Syntax/Parsing/ParsingErrorRecoveryTests.cs

```diff
@@ -6274,6 +6274,46 @@ class C
             Assert.Equal((int)ErrorCode.ERR_IdentifierExpected, file.Errors()[0].Code);
         }
 
+        [Fact]
```

Comment:
needs WorkItem attribute.

Reply:
What are the parameters to the WorkItem attribute?

Reply:
`WorkItem(24642, "https://github.com/dotnet/roslyn/issues/24642")`